internal/dag/policy.go

```diff
@@ -34,6 +34,22 @@ func retryPolicy(rp *projcontour.RetryPolicy) *RetryPolicy {
 	}
 }
 
+func headerPolicy(policy *projcontour.HeaderPolicy) *HeaderPolicy {
+	if policy == nil {
+		return nil
+	}
+
+	for _, rp := range policy.ReplaceHeader {
+		// Only host rewrite is currently supported
+		if rp.Key == "Host" {
```

Comment:
Could you use https://godoc.org/strings#EqualFold here to compare the header key case insenstively?

Reply:
👍